OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -207,76 +664,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="left">13</dim>
                                         <dim type="intercolumn">16</dim>
                                     </dimensions>
+                                    <note> There are 12-13 characters per lineؤ.</note>
+                                    <ab type="punctuation" subtype="Dividers">
+                                        The word <foreign xml:lang="gez">ቁም፡</foreign> is written at the end of the line on <locus target="#127vb"/>.
+                                    </ab>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.</ab>                         
                                 </layout>
                             </layoutDesc>
-                            
                         </objectDesc>
                         
+                        <handDesc>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus from="124ra" to="130va"/>
+                                <seg type="script"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1650" notAfter="1800"></date>
+                                <seg type="script"><foreign xml:lang="gez">Gʷǝlḥ</foreign>-script.</seg>
+                                <desc>Fine and trained hand. Characters are regularly spaced.</desc>                               
+                                <seg type="rubrication">
+                                    Incipits of few chapters and of the postscript; holy names; the word <foreign xml:lang="gez">ቁም፡</foreign>, written in a later 
+                                    hand on <locus target="#127vb"/>; elements of the numerals and of the punctuation signs, including text dividers.
+                                </seg>
+                                <list type="abbreviations">
+                                    <item>
+                                        <abbr>ምዕ፡</abbr> for <expan>ምዕራፍ፡</expan> (e.g., <locus target="#128ra #128va"/>)
+                                    </item>
+                                </list>
+                            </handNote>
+                        </handDesc>
                   
-                        
+                        <additions>
+                            <list>
+                                <item xml:id="e6">
+                                    <desc>Aids for identifying readings concerning the crucifixion of Jesus are written in the upper margin in deep vermilion ink: 
```

Comment:
If these and the readings indicated in the first msPart belong to one act of production, I would encode them in the physDesc pertaining to the entire manuscript. What do you think?

Reply:
Thank you for this remark. In fact, it's difficult to say whether all aids belong to a single act of production after the merging of the two msParts or not. They were probably added at different times by different hands, but who knows when? Those in msPart2 are all in deep vermillion ink, while the corresponding readings in msPart1 concerning the passion of Christ are written in the main hand (I have also added `corresp="#h1"` for clarity). This is why I suspect that they were done at different times. Nonetheless, there are some additional aids in the upper margin of msPart1 that are in vermillion ink (and other inks as well): it is possible that all the vermillion writings were made all together after the two msParts had been sewn together, but it's just hypothetical.